src/js/components/FlyoutMenu/ColumnMenu.jsx

```diff
@@ -0,0 +1,37 @@
+import PropTypes from 'prop-types'
+import React, { Fragment } from 'react'
+import { Popover, Transition } from '@headlessui/react'
+import { Icon } from '../Icon/Icon'
+
+function ColumnMenu({ children }) {
+  return (
+    <Popover className="relative mt-1">
+      {() => (
+        <>
+          <Popover.Button className="group inline-flex items-center rounded-md bg-white text-base font-medium hover:text-gray-900 focus:outline-none focus:ring-2 focus:ring-indigo-500 focus:ring-offset-2">
+            <Icon icon="fas columns" className="cursor-pointer text-gray-600" />
+          </Popover.Button>
+
+          <Transition
+            as={Fragment}
+            enter="transition ease-out duration-150"
+            enterFrom="opacity-0 translate-y-1"
+            enterTo="opacity-100 translate-y-0"
+            leave="transition ease-in duration-100"
+            leaveFrom="opacity-100 translate-y-0"
+            leaveTo="opacity-0 translate-y-1">
+            <Popover.Panel className="absolute -left-56 right-0 z-10 mt-3 w-screen max-w-md -translate-x-1/2 transform px-2 sm:px-0">
+              <div className="overflow-hidden bg-white rounded-lg shadow-lg ring-1 ring-black ring-opacity-5">
+                {children}
+              </div>
+            </Popover.Panel>
+          </Transition>
+        </>
+      )}
+    </Popover>
+  )
+}
+ColumnMenu.propTypes = {
+  children: PropTypes.node.isRequired
+}
+export { ColumnMenu }
```

Comment:
Is this used anywhere?

Reply:
I rewrote history to remove this from the commit. I accidentally had this on another branch.